h/views/helpers.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
```

Comment:
I gather from a brief search that this will become redundant in Python 3.14+ ?

Reply:
Yep, this is here to allow me to use names that're imported in `if TYPE_CHECKING:` blocks without having to put them in string quotes, by importing Python 3.14's deferred evaluation of annotations: https://docs.python.org/3.14/whatsnew/3.14.html#whatsnew314-pep649. More generally whenever you want to use a name in a type annotation when that name has not been imported or before that name has been declared, you can now do so without having to wrap the type name in quotes.

Unfortunately I don't know whether any of our tools (Ruff or Mypy) will point out when this import is no longer needed.

Of course `from __future__ import annotations` is used for lots of things so by the time we get to Python 3.14 who knows this file may want it for something else.